flink-runtime/src/test/java/org/apache/flink/runtime/rpc/akka/AkkaRpcActorTest.java

```diff
@@ -466,6 +466,23 @@ public void terminationFutureDoesNotBlockRpcEndpointCreation() throws Exception
 		}
 	}
 
+	@Test
+	public void resolvesRunningAkkaRpcActor() throws Exception {
+		final String endpointName = "foobar";
+
+		try (SimpleRpcEndpoint simpleRpcEndpoint1 = new SimpleRpcEndpoint(akkaRpcService, AkkaRpcServiceUtils.createRandomName(endpointName));
+			SimpleRpcEndpoint simpleRpcEndpoint2 = new SimpleRpcEndpoint(akkaRpcService, AkkaRpcServiceUtils.createRandomName(endpointName))) {
```

Comment:
```suggestion
		try (RpcEndpoint simpleRpcEndpoint1 = createRpcEndpointWithRandomNameSuffix(endpointNamePrefix);
			RpcEndpoint simpleRpcEndpoint2 = createRpcEndpointWithRandomNameSuffix(endpointNamePrefix)) {
```

Reply:
`HighAvailability#getJobManagerLeaderRetriever(JobID)` for which the address of the `StandaloneLeaderRetrievalService` is relevant will only be used by the `JobLeaderIdService` which is only interested in the leader session id and not the address. That's why we can change it.